docs/localization/improving-store-performance-with-preact.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Improving store performance with Preact"
+slug: "improving-store-performance-with-preact"
+hidden: false
+createdAt: "2024-10-31T18:10:15.623Z"
+updatedAt: ""
+---
+
+To reduce the package size used in the project and improve performance, you can enable [Preact](https://preactjs.com/) in your FastStore project. Preact is a lightweight alternative to [React](https://react.dev/) that improves performance by reducing the size of the JavaScript bundle. In a FastStore project, enabling Preact decreases the overall build size, resulting in faster page load times and better performance.
+
+> ⚠️ This is an experimental feature.
+
+| Before Preact | After Preact |
+| ------------------- | ------------ |
+| ![before-preact](https://vtexhelp.vtexassets.com/assets/docs/src/before-preact___aa7837e2280b911dd89fe3df3171f298.png) | ![after-preact](https://vtexhelp.vtexassets.com/assets/docs/src/after-preact___d5254919671e8304591421d3ec542610.png) |
+| The initial load times for the store's pages varied significantly, with some pages requiring up to 159 KB of JavaScript to load initially. This resulted in slower performance and longer wait times for users. Additionally, the shared JavaScript bundle was 94.1 KB, further contributing to the initial load size. | Preact optimized the store's performance by reducing the initial JavaScript load size for all pages. While some pages still required around 100-120 KB of JavaScript, this is an improvement compared to the previous 159 KB. The shared JavaScript bundle was also reduced to 58.7 KB. These optimizations led to faster load times and overall better performance for the store. |
+
+## Before you begin
+
+Make sure the `@faststore/cli` package from your project is updated to the latest version. To do so, follow the instructions in [Updating the '@fastore/cli' package version](https://developers.vtex.com/docs/guides/faststore/project-structure-updating-the-cli-package-version).
```

Comment:
In this case, the `cli` spelling is correct because it references the package name. Maybe we should add a guideline for packages' names to the style-check action? We would add each package's name to the guidelines, though. Or ignore the recommendation in this case.

cc: @anabaarbosa @carolinamenezes